tests/unit/mixins/data-adapter-mixin-test.js

```diff
@@ -116,10 +116,6 @@ describe('DataAdapterMixin', () => {
 
           expect(sessionService.invalidate).to.have.been.calledOnce;
         });
-
-        it('returns true', () => {
-          expect(adapter.handleResponse(401)).to.be.true;
-        });
```

Comment:
Done. I decided to use status 401 on the test for the return value as it represents the only special case in `handleResponse`. Any status code would return the same result.